fixtures/room/create_room_fixtures.py

```diff
@@ -280,3 +280,31 @@
             "createRoom": null
         }
     }
+
+increment_total_room_feedback_mutation = '''
```

Comment:
Is it increment total room feedback or total room feedback views?

Reply:
These changes are part of the old implementation so let me remove them since they are not part of the new implementation